cpp/tests/utility/Eigen.cpp

```diff
@@ -227,9 +225,7 @@ TEST(Eigen, ComputeJTJandJTr_vector) {
             [&](int i,
                 std::vector<Eigen::Vector6d, utility::Vector6d_allocator> &J_r,
                 std::vector<double> &r) {
-#ifdef _OPENMP
 #pragma omp critical
```

Comment:
Same, remove this `#pragma omp critical` .

Reply:
Done